src/templates/deployment.yaml

```diff
@@ -45,3 +45,13 @@ spec:
           env:
             - name: EULA
               value: {{ .Values.eula | squote }}
+          livenessProbe:
+            exec:
+              command: ["mc-monitor", "status"]
+            periodSeconds: 5
+            timeoutSeconds: 3
```

Comment:
To be honest I now question the timeout of 3 seconds.
I know I'm the one introducing it at OCI image level.
But I'm no longer aligned with this choice.
1 second should be enough.
Let's go for that (thus just remove the property).
If we observe unexpected restarts then we can always review and adjust the default later on.
Let's take the risk of being a bit "naive".